docs/src/app/pages/Form/Picklist/index.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import exampleDefaultCode from '!raw!./ExampleDefault';
+import inputCode from '!raw!react-lds/components/Form/Picklist';
+
+import CodeExample from '../../../components/CodeExample';
+import PropTypeDescription from '../../../components/PropTypeDescription';
+
+import ExampleDefault from './ExampleDefault';
+
+const mapId = (id) => {
+  let Component = ExampleDefault;
+  let componentCode = exampleDefaultCode;
+
+  switch (id) {
+    case 'default':
+      Component = ExampleDefault;
+      componentCode = exampleDefaultCode;
+      break;
+    default:
+      break;
+  }
```

Comment:
As I see it, this switch statement has no effect.

Reply:
yep, I left it there because we could later add the different states of the picklist there (required, multi, single...).